rclpy/src/rclpy_common/src/common.c

```diff
@@ -182,3 +182,119 @@ rclpy_convert_to_py(void * message, PyObject * pyclass)
   }
   return convert(message);
 }
+
+PyObject *
+rclpy_convert_to_py_subscription_event(void * callbackinfo, rcl_subscription_event_type_t event_type)
+{
+  PyObject * pyqos_event_module = PyImport_ImportModule("rclpy.qos_event");
+  if (!pyqos_event_module) {
+    return NULL;
+  }
+
+  PyObject * pyqos_event_class = NULL;
+  PyObject * pyqos_event_object = NULL;
+  if (event_type == RCL_SUBSCRIPTION_REQUESTED_DEADLINE_MISSED) {
+    pyqos_event_class = PyObject_GetAttrString(pyqos_event_module,
+      "QoSRequestedDeadlineMissedInfo");
+    Py_DECREF(pyqos_event_module);
+    if (!pyqos_event_class) {
+      return NULL;
+    }
+    rmw_requested_deadline_missed_status_t * actual_event =
+      (rmw_requested_deadline_missed_status_t *)callbackinfo;
+    PyObject * args = PyTuple_New(2);
+    if (!args) {
+      return NULL;
+    }
+    PyTuple_SET_ITEM(args, 0, PyLong_FromLong(actual_event->total_count));
```

Comment:
is this pattern safe? probably not - probably leaking refs - probably need to do error check and DECREF for every `PyLong_FromLong`